sqlx-core/src/pool/connection.rs

```diff
@@ -129,6 +129,13 @@ impl<DB: Database> PoolConnection<DB> {
     }
 }
 
+#[cfg(feature = "any")]
+impl PoolConnection<crate::any::Any> {
+    pub fn kind(&self) -> crate::any::AnyKind {
+        self.pool.connect_options.kind()
+    }
+}
+
```

Comment:
```suggestion
```

This isn't necessary because `PoolConnection` dereferences to `AnyConnection`, so calling `.kind()` will forward to `AnyConnection::kind()`.

Reply:
Done